css-transitions-1/Overview.bs

```diff
@@ -432,54 +432,6 @@ Starting of transitions {#starting}
         </span>
       </p>
 
-      <p>
-        Implementations must also maintain a set of
-        <dfn export lt="completed transition">completed transitions</dfn>,
-        each of which
-        (like <a>running transitions</a>)
-        applies to a specific element and non-shorthand property.
-        <span class="note">
-          This specification maintains the invariant that
-          there is never both a <a>running transition</a> and
-          a <a>completed transition</a> for the same property and element.
-        </span>
-      </p>
-
-      <p>
-        If an element is no longer in the document,
-        implementations must <a>cancel</a> any <a>running transitions</a>
-        on it and remove transitions on it from the
-        <a>completed transitions</a>.
-      </p>
```

Comment:
You should leave the rule that implementations cancel running transitions when an element is removed from the document (so leave the sentence except for the "and" and what follows).

Reply:
Oops, absolutely. Thanks.